test.js

```diff
@@ -32,13 +32,13 @@ const CSVHeader = CSVConfig.headers.map(i => i.name).join(';');
 
 const CSVInvalidFile = [
 	CSVHeader,
-	'Vasyl;Stokolosa;v.stokol@gmail.com;123;admin,manager',
-	'Vasyl_2;"";v.stokol@gmail.com;123123123;user',
+	'Vasyl;Stokolosa;v.stokol@gmail.com;123;admin,manager;',
```

Comment:
Wondering why you removed semicolons?

Reply:
Actually, I added semicolons. The delimiter is still required between Roles and Country columns even though the country value is optional. Otherwise, the structure is broken.

Reply:
If you download a .csv file from Drive the structure will be.
```js
Make,Style,Year,List
Audi,Van,2000,"1,2,3"
Ford,SUV,,1
```
Which .csv structure do you use in the project?

Reply:
We use a similar file structure but with a semicolon delimiter. And the example you provided is absolutely valid.

In the test examples, on the other hand, we have the following CSV:
```
First Name;Last Name;Email;Password;Roles;Country
Vasyl;Stokolosa;v.stokol@gmail.com;123;admin,manager
Vasyl_2;"";v.stokol@gmail.com;123123123;user
```
The last column is missing. That is why I wrote that the structure is broken without the last semicolon. Should be:
```
First Name;Last Name;Email;Password;Roles;Country
Vasyl;Stokolosa;v.stokol@gmail.com;123;admin,manager;""
Vasyl_2;"";v.stokol@gmail.com;123123123;user;""
```
or simply:
```
First Name;Last Name;Email;Password;Roles;Country
Vasyl;Stokolosa;v.stokol@gmail.com;123;admin,manager;
Vasyl_2;;v.stokol@gmail.com;123123123;user;
```
That is why I added the semicolon.

Reply:
Cool. I will release a new version by tomorrow.
Thanks for contributing 👍